Prac07/languages.py

```diff
@@ -0,0 +1,19 @@
+from Prac07.programminglanguage import ProgrammingLanguage
+
+def main():
+    # ruby = ProgrammingLanguage("Ruby","Dynamic",True,1995)
+    # python = ProgrammingLanguage("Python","Dynamic",True,1991)
+    # vb = ProgrammingLanguage("Visual Basic","Static",False,1991)
+
+    # print(ruby)
+
+    languages = [ProgrammingLanguage("Ruby","Dynamic",True,1995),
+                 ProgrammingLanguage("Python","Dynamic",True,1991),
+                 ProgrammingLanguage("Visual Basic","Static",False,1991)]
```

Comment:
Everything looks good for efficiency, however, for formatting, there are a number of times that "PEP 8" picks something up, they are related to the variable "languages" (most to do with "missing white space after ','").